python/katana/local/analytics/_ksssp.pyx

```diff
@@ -192,18 +222,116 @@ def ksssp(pg, str edge_weight_property_name, size_t start_node,
         katana.local.initialize()
 
         graph = Graph(get_rdg_dataset("ldbc_003"))
-        from katana.local.analytics import ksssp
+        from katana.local.analytics import ksssp, KssspStatistics
         weight_name = "workFrom"
         start_node = 0
         report_node = 10
         num_paths = 5
-        ksssp(graph, weight_name, start_node, report_node, num_paths)
+        table = ksssp(graph, weight_name, start_node, report_node, num_paths)
+        stats = KssspStatistics(graph, weight_name, table, report_node)
+        print(stats)
 
     """
 
     cdef string edge_weight_property_name_str = bytes(edge_weight_property_name, "utf-8")
     txn_ctx = txn_ctx or TxnContext()
-    with nogil:
-        handle_result_void(Ksssp(underlying_property_graph(pg), edge_weight_property_name_str,
-                                 start_node, report_node, num_paths, is_symmetric,
-                                 underlying_txn_context(txn_ctx), plan.underlying_))
+    return handle_result_ksssp(Ksssp(underlying_property_graph(pg), edge_weight_property_name_str,
+                                  start_node, report_node, num_paths, is_symmetric,
+                                  underlying_txn_context(txn_ctx), plan.underlying_))
+
+
+cdef _KssspStatistics handle_result_KssspStatistics(Result[_KssspStatistics] res) nogil except *:
+    if not res.has_value():
+        with gil:
+            raise_error_code(res.error())
+    return res.value()
+
+
+cdef class PathStats:
+    cdef _PathStats underlying
+
+    @staticmethod
+    cdef PathStats make(_PathStats u):
+        f = <PathStats>PathStats.__new__(PathStats)
+        f.underlying = u
+        return f
+
+    @property
+    def path(self) -> List[int]:
+        """
+        Path from a source to destination node
+
+        :rtype: List[int]
+        """
+        res = []
+        for i in range(self.underlying.path.size()):
+            res.append(self.underlying.path.at(i))
+
+        return res
```

Comment:
NIT: I think you can just return `self.underlying.path` and Cython will just convert it. But it doesn't matter this will all get discarded once we move to pybind11.

Reply:
Tried returning self.underlying.path at [this commit](https://github.com/KatanaGraph/katana/blob/56743a16f4703196ca1478da32ef1e6a59a7ca32/python/katana/local/analytics/_ksssp.pyx) but it failed to compile due to not automatically converting, so had to do this